android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginActivity.kt

```diff
@@ -46,7 +46,7 @@ class LoginActivity :
     loginViewModel.apply {
       navigateToHome.observe(this@LoginActivity, { loginService.navigateToHome() })
       launchDialPad.observe(this@LoginActivity, { if (!it.isNullOrEmpty()) launchDialPad(it) })
-      loginUser()
+      // loginUser() commented out to make user login everytime
```

Comment:
why is this the desired functionality?

Reply:
@dubdabasoduba requested this change to ensure that in field user always login when opening app

Reply:
@pld leaving it to always auto authenticate might lead to infringement of data privacy while on live usage.

Reply:
Gotcha, makes sense, should we add a todo here to make this configurable on a user or app bases, would that be useful?

Reply:
added todo

Reply:
cool thanks